app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,82 @@
+class MoviesController < ApplicationController
+  include ApplicationHelper
+
+  def index
+    render 'index'
+  end
+
+
+  def show
+    @movie = Movie.find_by(id: params[:id])
+    @comments = @movie.comments
+    @conversation = Conversation.new
+    activity_logger('movie', @movie) if @movie
+    @comment = Comment.new
+    @related_movies = @movie.related_movies
+
+  end
+
+  def search
+    if request.xhr?
+      @movies = get_movies_by_title(params["searchResult"])
+    else
+      @movies = get_movies_by_title(params['search_form'])
+    end
+
+    if @movies
+      @movies.map! do |movie|
+        movie = Movie.find_by(imdbID: movie.imdbID)
```

Comment:
No need to reassign `movie` here, since
```ruby
Movie.find_by(imdbID: movie.imdbID)
```
 will return the exact same value as
```ruby
movie = Movie.find_by(imdbID: movie.imdbID)
```